tests/build_/test_build_request.py

```diff
@@ -954,7 +954,7 @@ def test_render_prod_request_requires_newer(self):
         ['v2'],
     ])
     @pytest.mark.parametrize('platform', [None, 'x86_64'])
-    @pytest.mark.parametrize('arrangement_version', [3, 4])
+    @pytest.mark.parametrize('arrangement_version', [3, DEFAULT_ARRANGEMENT_VERSION])
```

Comment:
Instead of `[3, DEFAULT_ARRANGEMENT_VERSION]`, should this be: `range(3, DEFAULT_ARRANGEMENT_VERSION+1)` ?

That way when we up to arrangement 5, 4 is not left untested.

Reply:
I think range(3, DEFAULT_ARRANGEMENT) since we shouldn't generally test an arrangement greater than DEFAULT which is normally the highest available. I'll make the change.

Reply:
The limit of the `range` function is non-inclusive thus the `+1`:
```python
>>> range(3, 4)
[3]
>>> range(3, 4+1)
[3, 4]
```